web/vtadmin/src/components/routes/Workflows.tsx

```diff
@@ -70,6 +72,16 @@ export const Workflows = () => {
                 <tr key={idx}>
                     <DataCell>
                         <div className="font-bold">{href ? <Link to={href}>{row.name}</Link> : row.name}</div>
+                        {row.workflowType && (
+                            <div className="text-secondary text-success-200">
+                                {row.workflowType}
```

Comment:
For this one we'll still want to handle the null case from [line 57](https://github.com/vitessio/vitess/pull/11685/files#diff-654f842b149e2f895f03d04f85c065655325acfc3d045c19db272c5e69871ebaR57) 😄 
```suggestion
                                {row.workflowType || 'N/A'}
```

Reply:
> For this one we'll still want to handle the null case from [line 57](https://github.com/vitessio/vitess/pull/11685/files#diff-654f842b149e2f895f03d04f85c065655325acfc3d045c19db272c5e69871ebaR57) smile

I thought it would never reach this point (line 77) if `row.workflowType` is null, because of the check on line 75?

Reply:
Ah I missed that! You're correct. This is great!